packages/todos-lib/package.js

```diff
@@ -70,4 +70,19 @@ Package.onUse(function(api) {
   api.imply([
     'method',
   ]);
+
+  // i18n
+  api.imply([
+    'tap:i18n@1.7.0'
+  ]);
+
+  // App libs
+  api.use([
+    // TODO -- I need to add this because of a bug in tap:18n I guess?
```

Comment:
Like outside of packages? I guess we could try and do that.

Then you would have a situation where a package calls a symbol like "Lists.methods.remove.lastPublicList" and doesn't define even the english translation of it which seems a bit weird though.

I'm not really sure why but all you need to do is add 

```js
api.addFiles('package-tap.i18n');
```

(with an empty file)

to be able to use `__` in your package, and then the actual `i18n.json` files themselves to add translations. I'm not sure it's *that* intense?